src/alias.c

```diff
@@ -39,7 +39,7 @@ char *replace_alias(char *cmd, alias_t *alias)
 
     if (!arg_array)
         return (NULL);
-    for (int i = 0; arg_array[i]; i++) {
+    for (int i = 0; i == 0; i++) {
```

Comment:
Just to make sure, is it just because we have to replace first arg of each command?

Reply:
The alias.c has been totaly reworked with the new parser to prevent multiples spaces/tabs to dissapear. Changes to this file won't be reflected on the master.